symfony/templates/participant/details.html.twig

```diff
@@ -497,7 +497,6 @@
                                                 {{ form_start(patientStatusForm, { attr: { class: 'warn-unsaved disable-enter prevent-resubmit patient-status-form' } }) }}
                                                 {{ form_errors(patientStatusForm) }}
                                                 {{ form_rest(patientStatusForm) }}
-                                                {{ form_widget(patientStatusForm.comments) }}
```

Comment:
Do you know why this was here before? Were we trying to render the comments field differently?

Reply:
I don't think we are rendering the comments field differently. I think `form_widget` was added by mistake.

Reply:
If I recall correctly, any fields you try to render _after_ calling `{{ form_rest( ... ) }}` on L499 will never render because they have been marked as already rendered.